java-compiler-testing/src/test/java/io/github/ascopes/jct/testing/unit/compilers/SimpleCompilerTest.java

```diff
@@ -74,6 +75,26 @@
 @DisplayName("SimpleCompiler tests")
 class SimpleCompilerTest {
 
+  @DisplayName("AlreadyUsedCompilerException is thrown on second call to compile()")
+  @Test
+  void alreadyUsedCompilerExceptionIsThrown() {
+	  try (var compilationFactory = mockConstruction(SimpleCompilationFactory.class)) {
+	    // Given
+	    var jsr199Compiler = stub(JavaCompiler.class);
+	    var flagBuilder = stub(FlagBuilder.class);
+	   var compiler = new StubbedCompiler("foobar", jsr199Compiler, flagBuilder);
```

Comment:
Think checkstyle will raise the indent here. If you use IntelliJ you should be able to just import the project and run the "reformat file" command, it should pick up everything from the `.editorconfig` at the root of the repository.